packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -238,6 +237,29 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
   }
 
   @autobind
+  private _onRenderTextContents(): JSX.Element | (JSX.Element | null)[] {
+    let {
+      text,
+      children,
+      description,
+      onRenderText = this._onRenderText,
+      onRenderDescription = this._onRenderDescription
+    } = this.props;
+
+    if (this.props.text || typeof (children) === 'string' || this.props.description) {
+      return React.createElement(
+        'div' as any,
+        { className: this._classNames.textContainer },
+        onRenderText(this.props, this._onRenderText),
+        onRenderDescription(this.props, this._onRenderDescription));
+    }
```

Comment:
Why do we use React.creatElement here instead of JSX?

Reply:
This is following the convention for building the button that is already in place. Not sure what the reasoning is for why this was done with the existing code